Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -5,6 +5,8 @@ import Foundation
 
 class ManageOfflineRegionsExample_Swift: UIViewController, MGLMapViewDelegate {
 
+    let inBytes = ByteCountFormatter()
```

Comment:
I accidentally led you astray with this suggestion, I'm sorry! 

So, it looks like the `ByteCountFormatter.string(fromByteCount:countStyle:)` is a _class_ method. This means we don't need to initialize a `ByteCountFormatter` to call this specific method:

For example,

```swift
ByteCountFormatter().string(fromByteCount:countStyle:) // won't work
ByteCountFormatter.string(fromByteCount:countStyle:) // works
```

Because of this, there actually isn't a need to hold a reference to a `ByteCountFormatter` here, because we don't need an instance of the `ByteCountFormatter` class to call the method we want to use. 

In this file you're using two different methods on `ByteCountFormatter`. One is the class function (or class method), and the other is an instance method:

```swift
// Within offlinePackProgressDidChange
// Class method - called on the type itself
let byteCount = ByteCountFormatter.string(fromByteCount: Int64(pack.progress.countOfBytesCompleted), countStyle: ByteCountFormatter.CountStyle.memory)

// Within cellForRowAt:indexPath
// Instance method - called on an instance of the type
cell.textLabel?.text = "Region \(indexPath.row + 1): size: \(inBytes.string(fromByteCount: Int64(pack.progress.countOfBytesCompleted)))"
```

Here's a suggestion - how about we add a second computed property on `MGLOfflinePackProgress`:

```swift
var formattedCountOfBytesCompleted: String {
    return ByteCountFormatter.string(fromByteCount: Int64(countOfBytesCompleted),
                                     countStyle: .memory)
}
```

And then the updates will look like this:

```swift
// Within offlinePackProgressDidChange
let byteCount = pack.progress.formattedCountOfBytesCompleted

// Within cellForRowAt:indexPath
cell.textLabel?.text = "Region \(indexPath.row + 1): size: \(pack.progress.formattedCountOfBytesCompleted)"
```